test-framework/test-suites/integration/tests/report/test_report_host.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+
+@pytest.mark.usefixtures("revert_database")
+class TestReportHost:
+    def test_report_host_with_default_network_behaviour():
+        result = host.run(f'stack add host backend-0-0')
```

Comment:
No need to make things 'f' strings if you aren't injecting any formatting info. It doesn't hurt anything, but plan old strings will be most likely slightly faster.

Reply:
(and other developers won't double-check the string trying to figure out where the variable is :)